crates/evm/fuzz/src/strategies/state.rs

```diff
@@ -100,10 +101,12 @@ pub struct FuzzDictionary {
     addresses: AIndexSet<Address>,
     /// Configuration for the dictionary.
     config: FuzzDictionaryConfig,
-    /// New key indexes added to the dictionary since container initialization.
-    new_values: Vec<usize>,
-    /// New address indexes added to the dictionary since container initialization.
-    new_addreses: Vec<usize>,
+    /// Number of state values initially collected from db.
+    /// Used to revert new collected values at the end of each run.
+    db_state_values: usize,
+    /// Number of address values initially collected from db.
+    /// Used to revert new collected addresses at the end of each run.
+    db_addreses: usize,
```

Comment:
typo
```suggestion
    db_addresses: usize,
```

Reply:
changed in https://github.com/foundry-rs/foundry/pull/8080/commits/a5db0bc34e98858d3f19adb3ecb032dca2632a75